content/en/docs/collector/configuration.md

```diff
@@ -513,54 +517,56 @@ service:
-receiver(s)/exporter(s) referenced in multiple pipelines, where only one
-instance of a receiver/exporter is used for all pipelines. Also note that the
-order of processors dictates the order in which data is processed.
+You can use the same receiver, processor, or exporter in more than one pipeline.
+When a processor is referenced in multiple pipelines, each pipeline gets a
+separate instance of the processor.
 
-The following is an example pipeline configuration:
+The following is an example of pipeline configuration. Note that the order of
+processors dictates the order in which data is processed:
 
 ```yaml
 service:
   pipelines:
     metrics:
       receivers: [opencensus, prometheus]
+      processors: [batch]
       exporters: [opencensus, prometheus]
     traces:
       receivers: [opencensus, jaeger]
-      processors: [batch]
+      processors: [batch, memory_limiter]
       exporters: [opencensus, zipkin]
 ```
 
 ### Telemetry
 
-Telemetry is where the telemetry for the collector itself can be configured. It
-has two subsections: `logs` and `metrics`.
+You can configure telemetry for the Collector itself in the `telemetry`
+subsection inside the `service` section. Collector telemetry can be useful when
+troubleshooting Collector issues.
 
-The `logs` subsection allows configuration of the logs generated by the
-collector. By default the collector will write its logs to stderr with a log
-level of `INFO`. You can also add static key-value pairs to all logs using the
-`initial_fields` section.
-[View the full list of `logs` options here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L41-L97)
+The `logs` subsection lets you configure logs generated by the Collector. By
+default, the Collector writes its logs to stderr with a log level of `INFO`. You
+can also add static key-value pairs to all logs using the `initial_fields`
+section. See the full list of `logs` options
+[here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L41-L97)
 
-The `metrics` subsection allows configuration of the metrics generated by the
-collector. By default the collector will generate basic metrics about itself and
-expose them for scraping at `localhost:8888/metrics`
-[View the full list of `metrics` options here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111)
+The `metrics` subsection lets you configure metrics generated by the Collector.
+By default, the Collector generates basic metrics about itself and expose them
+for scraping at <http://localhost:8888/metrics>. See the full list of options
+for the `metrics` subsection
+[here.](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111)
```

Comment:
As I mentioned in the other PR making changes to this section, I don't think we should be linking out to the source file. Especially because that [**config file has been deprecated**](https://github.com/open-telemetry/opentelemetry-collector/blob/50f013d0d242bf7b600eddf5cfe484e087c1de0c/CHANGELOG.md?plain=1#L1237)!

We should just list the "full options" (there aren't that many) here.

---

If you choose to keep the link for now, I'd at least suggest the following rewording to avoid a "here" link:

```suggestion
for scraping at <http://localhost:8888/metrics>. For the full list of options
for the `metrics` subsection, see the 
[`ServiceTelemetryMetrics struct`](https://github.com/open-telemetry/opentelemetry-collector/blob/7666eb04c30e5cfd750db9969fe507562598f0ae/config/service.go#L99-L111) comments.
```

Reply:
@theletterf et al.: I've created a followup issue to track further improvements to this point:

- #3514